s3_select_executor/README.md

```diff
@@ -1,16 +1,25 @@
 # Introduction
 
-## Here are the steps to do before you run the script:
+We realized that using S3 select we can run a lot of validations and also run ad-hoc filtering on the data available in S3.
+
+The major use case which got resolved from this script is data validation and thus has helped us do this in a cost effective manner.
+
+
+
+# Requirements to run this script
 
 * Install Python on Windows (https://www.python.org/downloads/release/python-2715/) / Ubuntu users would have it already installed(version 2.7).
 * Install Pip on Windows (https://github.com/BurntSushi/nfldb/wiki/Python-&-pip-Windows-installation) / Ubuntu users would do using sudo apt-get install pip
 * Install AWS CLI on Windows (https://docs.aws.amazon.com/cli/latest/userguide/awscli-install-windows.html) / Ubuntu users would do using pip install awscli --upgrade
 * Install Boto using: pip install boto3
 * Run the following to configure aws: aws configure
 
-Once the pre-requisits are installed, then starts the excution.
 
 
+# Execution
+
+Once the pre-requisits are installed, then starts the excution.
```

Comment:
Small typo in pre-requisits. I guess it should be prerequisites.(Once the prerequisites are installed, start the execution)

Reply:
Wow, thanks, will change it

Reply:
Done